src/Service/CsvImport.php

```diff
@@ -68,4 +74,77 @@ public function createSeason(string $seasonName): Season
         }
         return $season;
     }
+
+    public function createSubscribers(array $csvData): int
+    {
+        $total = 0;
+
+        foreach ($csvData as $subscription) {
+            // Recherche du subscriber par numéro de licence
+            $subscriber = $this->subscriberRepository->findOneBy(['licenceNumber' => $subscription['NO ADHERENT']]);
+
+            if ($subscriber == null) {
+                $subscriberBirthdate = explode('/', $subscription['DATE NAISSANCE']);
+
+                // Si le numéro de licence n'existe pas, recherche par nom/prénom/date de naissance (cas licence D)
+
+                $subscriber = $this->subscriberRepository->findOneBy([
```

Comment:
ok c'est bien, mais tu peux optimiser
                 $subscriber = $this->subscriberRepository->findOneBy licence ??  $this->subscriberRepository->findOneBy nom/prenom/datenaissance ?? new Subscriber();

Et ensuite, tu fais tes setXXX() => ça t'évite de répeter la partie hydratation.